simapp/ante.go

```diff
@@ -0,0 +1,60 @@
+package simapp
```

Comment:
Now that this file is almost a duplication of x/auth/ante, is there a way to simplify of the two to reduce duplication?

Reply:
unless auth depends on circuit there is no way. This is what most apps do because of ibc anyways

Reply:
Yes, but I was thinking of possibly having auth's `NewAnteHandler` changed into a helper returning `[]sdk.AnteDecorator`, so that apps have less to configure in their own `NewAnteHandler`.
This may be tricky when needing a specific order 🤔

Reply:
This was something I attempted earlier on but was creating more issues. You can see earlier comments from Aaron and Bez

Reply:
This could be done in a follow-up anyway, given that it is more related to clean-up auth that circuit.
However, apart from the ordering that could become trickier (but possible), I am not understanding what other issues would happen (I haven't found the comments :))

Reply:
I completely read this wrong haha i was referring to this work [here](https://github.com/cosmos/cosmos-sdk/pull/14521/commits/f614387efdfeaef6677be9bc446ea51e2176e2f7#diff-549ae89228e2040814cba42d57360575faa1d51e1a018a4415e44434742e3622). What you suggest theoretically makes sense and am in favour of the experimentation. Would also be keen to tag on that